test/image-blueprints/layer4-art/group1/rhel92_microshift-art.toml

```diff
@@ -0,0 +1,39 @@
+name = "rhel-92-microshift-art-base"
+description = "A RHEL 9.2 image with the RPMs built from ART."
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-92"
+
+[[packages]]
+name = "microshift"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-networking"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "microshift-selinux"
+version = "{{ .Env.SOURCE_VERSION }}"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
+
```

Comment:
Can we add here these packages? They may be useful for troubleshooting. Also, `iproute-tc` is needed for Stress Testing.
I copy them from [blueprint_v0.0.1.toml](https://github.com/openshift/microshift/blob/main/scripts/image-builder/config/blueprint_v0.0.1.toml)
```
[[packages]]
name = "iputils"
version = "*"

[[packages]]
name = "bind-utils"
version = "*"

[[packages]]
name = "net-tools"
version = "*"

[[packages]]
name = "iotop"
version = "*"

[[packages]]
name = "strace"
version = "*"

[[packages]]
name = "iproute-tc"
version = "*"
```